apps/omg_child_chain/lib/omg_child_chain/ethereum_event_aggregator.ex

```diff
@@ -203,7 +203,8 @@ defmodule OMG.ChildChain.EthereumEventAggregator do
         {blknum, signature}
       end)
 
-    if :ets.insert(state.ets_bucket, data), do: :ok, else: {:error, :could_not_store_logs}
+    true = :ets.insert(state.ets_bucket, data)
```

Comment:
So raise an error instead of returning an error tuple, which will result (if I'm not mistaken) in the aggregator being restarted by its supervisor with proper re-initialization of the `ets` "relationship" - alright, pretty nice!